src/encoded/cart_view.py

```diff
@@ -1,14 +1,26 @@
 from pyramid.view import view_config
 from collections import OrderedDict
+from snovault import COLLECTIONS
+from pyramid.httpexceptions import HTTPBadRequest
```

Comment:
It's better to group "like" imports...so all pyramid or all the snovault imports are in the same block.  pep8 says to put a space between groups of imports, we don't have a pattern of leaving spaces, but keeping the organized into groups makes it more readable. 

Also, I didn't find this in pep8, but I'll usually leave two spaces between imports, top level vars, and the first function declaration.

Reply:
Changed to group them and add blank lines.